website/content/en/pre-docs/tasks/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Tasks"
+linkTitle: "Tasks"
+weight: 45
+---
+
+Karpenter tasks can be divided into those for a cluster operator who is managing the cluster itself and application developers who are deploying pod workloads on a cluster.
```

Comment:
I think "cluster operator" is ambitious. Is this an "operator" like a kubernetes object? or is this a person responsible for operating kubernetes?

Reply:
The wording was suggested by PM. I'll circle back with him and double-check that he wants that. I wouldn't mind "cluster administrator", but I think that's what it was changed from.

Reply:
DevOps is split into (kubernetes) developer and (kubernetes) operator